src/ibtracert.c

```diff
@@ -778,15 +778,84 @@ static int process_opt(void *context, int ch, char *optarg)
 	return 0;
 }
 
-int main(int argc, char **argv)
-{
-	int mgmt_classes[3] =
-	    { IB_SMI_CLASS, IB_SMI_DIRECT_CLASS, IB_SA_CLASS };
+static int get_route(char *srcid, char *dstid) {
 	ib_portid_t my_portid = { 0 };
 	ib_portid_t src_portid = { 0 };
 	ib_portid_t dest_portid = { 0 };
 	Node *endnode;
```

Comment:
These variables are no longer needed and can now be removed in main(), right ?

Reply:
They only reside in the function now, and are no longer in main.
The way git created the patch was kind of confusing. (IMO)

Not sure how to change the title...

Reply:
Oh, you're right. Of course, they're not part of the new main.

As far as updating the commit, you can amend it to update the description only and then push -f.

Reply:
git rebase -I starting at commit ID prior to this
then amend the first commit
then git push -f